plugins/out_stackdriver/stackdriver_http_request.c

```diff
@@ -185,16 +185,23 @@ void add_http_request_field(struct http_request_field *http_request,
 /* latency should be in the format:
  *      whitespace (opt.) + integer + point & decimal (opt.)
  *      + whitespace (opt.) + "s" + whitespace (opt.)
+ *
+ * latency is Duration, so the maximum value is "315576000000.999999999s".
+ * (23 characters in length)
  */
 static void validate_latency(msgpack_object_str latency_in_payload,
                              struct http_request_field *http_request) {
     flb_sds_t pattern = flb_sds_create("^\\s*\\d+(.\\d+)?\\s*s\\s*$");
-    char extract_latency[latency_in_payload.size];
+    char extract_latency[64];
```

Comment:
Since the possible maximum length is 23, could you use 32 instead of 64 here?

Reply:
Fixed in d5b388.